pq/publication/replica_identity.go

```diff
@@ -66,10 +66,17 @@ func (c *Publication) GetReplicaIdentities(ctx context.Context) ([]Table, error)
 	tableNames := make([]string, len(c.cfg.Tables))
 
 	for i, t := range c.cfg.Tables {
-		tableNames[i] = "'" + t.Name + "'"
+		if t.Schema == "" && !strings.Contains(t.Name, ".") {
+			tableNames[i] = "'" + t.Name + "'"
+		} else {
+			tableNames[i] = "'" + t.Schema + "." + t.Name + "'"
+		}
+
 	}
 
-	query := "SELECT relname AS table_name, relreplident AS replica_identity FROM pg_class WHERE relname IN (" + strings.Join(tableNames, ", ") + ")"
+	query := "SELECT relname AS table_name, n.nspname AS schema_name, relreplident AS replica_identity FROM pg_class c JOIN pg_namespace n ON c.relnamespace = n.oid WHERE concat(n.nspname, '.', c.relname) IN (" + strings.Join(tableNames, ", ") + ")"
```

Comment:
Can we consider using fmt.Sprintf here instead of concatenating strings directly? This could improve readability and reduce the risk of errors in constructing the PostgreSQL statement. What do you think?